c2corg_ui/static/js/documentediting.js

```diff
@@ -679,5 +683,46 @@ app.DocumentEditingController.prototype.setOrientation = function(orientation, d
 }
 
 
-app.module.controller('appDocumentEditingController',
-    app.DocumentEditingController);
+/**
+ * @param {Object} doc
+ * @export
+ */
+app.DocumentEditingController.prototype.pushToAssociations = function(doc) {
+  var routesLength =  this.scope_[this.modelName_]['associations']['routes'].length;
+  var associations = this.scope_[this.modelName_]['associations'];
+
+  if (doc['type'] === 'r') {
```

Comment:
I think that after we release the next alpha or beta version, we should create a special PR to clean up this mess and create controllers/factories for document-specific controls.